### ISSUE TITLE
Raycast Search Emojii & Symbols Floating Applied to Parent

### ISSUE THREAD

**Issue by felixge** [nikitabobko/AeroSpace#254]
## Steps to reproduce:

1. Setup Raycast to open it's `Search Emojii & Symbols` floating window via `Alt+Shift+Space`.
2. Open Chrome (or any other application)
3. Press `Alt+Shift+Space` to bring up the `Search Emojii & Symbols` window. (Note: Invoking Raycast itself first and then selecting `Search Emojii & Symbols` doesn't cause the issue, it has to be via the keyboard shortcut)

## Expected Behavior

The `Search Emojii & Symbols` should float, but the Chrome window (or whatever the parent window was) is not impacted.

## Actual Behavior

The parent window is being switched into floating mode. Restoring tiling mode via `layout floating tiling` which I have mapped to `alt+s f` fixes the situation. The video below shows the problem in action. I'm placing a terminal window next to Chrome in order to show the impact of the Chrome window going into float mode.

https://github.com/nikitabobko/AeroSpace/assets/15000/8e243408-71db-4619-b851-611334fe7d0b

Thank you so much for creating this project. Please let me know if this report makes sense or if I can provide more information.

**Comment by nikitabobko** [comment]
The issue sounds weird. Are you sure that you don't have `alt-shift-space` mapping in your `aerospace.toml` config?

I couldn't reproduce the bug by following your steps. The focused window stays unaffected

**Comment by felixge** [comment]
> The issue sounds weird. Are you sure that you don't have `alt-shift-space` mapping in your `aerospace.toml` config?

🤦🏻‍♂️ that was the problem. I removed the mapping and things are working great now.

> I couldn't reproduce the bug by following your steps. The focused window stays unaffected

Thank you so much for taking the time, and apologies for not realizing my mistake myself.

I've been trying out different tiling WMs for mac over the years and always ran into weird issues like this. But AeroSpace seems to be the first WM that actually works well enough to be my daily driver. Thank you so much for creating this project and your help here. I've made a small GH sponsorship donation as a token of my appreciation ❤️
